### PR TITLE
[TTT ] Fix radio calling out dead players

### PR DESCRIPTION
**Pull request by end360**
Fixes the radio calling out dead or spectating players.

### CODE REVIEW

**Comment by zerfgog** [comment]
might be neater if lines 261 and 262 were consolidated with
```lua
if ent:IsPlayer() and ent:IsTerror() then
```

**Comment by end360** [comment]
@zerfgog I assumed that if the entity Isn't a player then an error would be thrown if you tested if IsTerror() since I believe that works only with players

**Comment by bigdogmat** [comment]
The second condition is only checked if the first one is true.

**Comment by bigdogmat** [comment]
You forgot to undo the indention

**Comment by end360** [comment]
Thanks for telling me that, I'm a bit distracted by class right now.

File: garrysmod/gamemodes/terrortown/gamemode/cl_voice.lua [hunk 1]
```diff
@@ -258,7 +258,7 @@ function RADIO:GetTargetType()
 
    local ent = trace.Entity
 
-   if ent:IsPlayer() then
+   if ent:IsPlayer()and  ent:IsTerror() then
```
Comment (svdm) [inline garrysmod/gamemodes/terrortown/gamemode/cl_voice.lua]: Good fix and I hate to nitpick, but please make the `and` have one space on either side.

**Pull request merged**